lib-npx/installCopyFiles.js

```diff
@@ -41,23 +41,25 @@ function createPackageJson() {
     // This is the package.json contents that will be in the target directory
     const rootPackageJson = {
         name: 'iobroker.inst',
-        version: ownPackage.version,
+        version: '3.0.0',
         private: true,
         description: 'Automation platform in node.js',
         // Copy scripts and required engine from our own package.json
         scripts: {
             'install-service': 'node install.js',
             'uninstall-service': 'node uninstall.js'
         },
-        engine: ownPackage.engine,
+        engines: {
+            'node': '>=12.0.0'
+        },
         // Require the dependencies in our own package.json plus the following ones
-        dependencies: Object.assign({}, ownPackage.dependencies, {
+        dependencies: {
```

Comment:
intended from my pov yes because:
* the "former code" also added deps of the installer package (like fd-extra and such) to the main deps
* the "Linux installer" do not do it
So now it is "in sync" with the linux installer

Reply:
Then let’s remove the comment or adjust it 😬

Reply:
when you are right ...